src/app/pages/select-network/components/network-list-item.layout.tsx

```diff
@@ -1,3 +1,4 @@
+// TODO get icons for Trash, Circle, Star, Zap, CloudOff
```

Comment:
@fbwoolf  - these were the last few I was unsure of. Maybe we need to take them out of Figma. 

I couldn't see ones for Circle/ Star/ Zap/ CloudOff there